src/Testcontainers/Builders/TestcontainersBuilderAzuriteExtension.cs

```diff
@@ -0,0 +1,139 @@
+namespace DotNet.Testcontainers.Builders
+{
+  using System.Collections.Generic;
+  using System.Collections.ObjectModel;
+  using System.IO;
+  using DotNet.Testcontainers.Configurations.Modules.Databases;
+  using DotNet.Testcontainers.Containers;
+  using JetBrains.Annotations;
+
+  /// <summary>
+  /// This class applies the extended Testcontainer configurations for Azurite.
+  /// </summary>
+  [PublicAPI]
+  public static class TestcontainersBuilderAzuriteExtension
+  {
+    public static ITestcontainersBuilder<AzuriteTestcontainer> WithAzurite(this ITestcontainersBuilder<AzuriteTestcontainer> builder, AzuriteTestcontainerConfiguration configuration)
+    {
+      builder = builder
+        .WithImage(configuration.Image)
+        .WithEnvironment(new ReadOnlyDictionary<string, string>(configuration.Environments))
+        .WithOutputConsumer(configuration.OutputConsumer)
+        .WithWaitStrategy(configuration.WaitStrategy)
+        .ConfigureContainer(container =>
+        {
+          container.ContainerBlobPort = configuration.RunBlobOnly || configuration.RunAllServices ? AzuriteTestcontainerConfiguration.DefaultBlobPort : 0;
+          container.ContainerQueuePort = configuration.RunQueueOnly || configuration.RunAllServices ? AzuriteTestcontainerConfiguration.DefaultQueuePort : 0;
+          container.ContainerTablePort = configuration.RunTableOnly || configuration.RunAllServices ? AzuriteTestcontainerConfiguration.DefaultTablePort : 0;
+        });
+
+      if (configuration.RunBlobOnly || configuration.RunAllServices)
+      {
+        builder = builder.WithPortBinding(configuration.BlobPort, AzuriteTestcontainerConfiguration.DefaultBlobPort);
+      }
+
+      if (configuration.RunQueueOnly || configuration.RunAllServices)
+      {
+        builder = builder.WithPortBinding(configuration.QueuePort, AzuriteTestcontainerConfiguration.DefaultQueuePort);
+      }
+
+      if (configuration.RunTableOnly || configuration.RunAllServices)
+      {
+        builder = builder.WithPortBinding(configuration.TablePort, AzuriteTestcontainerConfiguration.DefaultTablePort);
+      }
+
+      if (!string.IsNullOrWhiteSpace(configuration.Location))
+      {
+        builder = builder.WithBindMount(configuration.Location, AzuriteTestcontainerConfiguration.DefaultLocation);
+      }
+
+      builder = builder
```

Comment:
Immediately return.

Reply:
Fixed in [bddb8be](https://github.com/testcontainers/testcontainers-dotnet/pull/555/commits/bddb8be4dc91b51a89a704799bbe006d784e4f8c)